### DISCUSSION TITLE
Add Kerberos support for postgres connection

### DISCUSSION THREAD

**Discussion by juneja-deshaw** [prest/prest#908] [API Server]
Currently `pREST` only supports username and password based authentication for connecting to the DB (https://docs.prestd.com/get-started/configuring-prest). Is there a way that we can also support GSS Auth here?
I see it uses `pq` library https://github.com/prest/prest/blob/main/go.mod#L13 and `pq` supports GSS Auth https://github.com/lib/pq.

**Comment by avelino** [comment]
> Currently `pREST` only supports username and password based authentication for connecting to the DB (https://docs.prestd.com/get-started/configuring-prest). Is there a way that we can also support GSS Auth here?
> I see it uses `pq` library https://github.com/prest/prest/blob/main/go.mod#L13 and `pq` supports GSS Auth https://github.com/lib/pq.
> 

Were you able to make it work?

**Comment by juneja-deshaw** [comment]
I opened an issue https://github.com/prest/prest/issues/909 for the same.
